src/main/java/liquibase/ext/snowflake/database/SnowflakeDatabase.java

```diff
@@ -84,6 +86,54 @@ public String getDefaultDriver(String url) {
         return null;
     }
 
+    @Override
+    public String getLiquibaseSchemaName() {
```

Comment:
I don't think we need any of these methods to be overriden do we? The default logic should be good as long as SnowflakeDatabase is correctly saying schema names are stored as uppercase?